mbutil/util.py

```diff
@@ -230,10 +236,8 @@ def disk_to_mbtiles(directory_path, mbtiles_file, **kwargs):
                             (?, ?, ?, ?);""",
                             (z, x, y, sqlite3.Binary(file_content)))
                         count = count + 1
-                        if (count % 100) == 0:
-                            for c in msg: sys.stdout.write(chr(8))
-                            msg = "%s tiles inserted (%d tiles/sec)" % (count, count / (time.time() - start_time))
-                            sys.stdout.write(msg)
+                        if (count % 100) == 0 and not silent:
+                            logger.info(" %s tiles inserted (%d tiles/sec)" % (count, count / (time.time() - start_time)))
```

Comment:
I'm ok with this change, makes things more consistent. I think the chances of breaking anything is minimal. I can't think why anyone would need to parse that particular output but maybe I'm not being creative enough :-)